trace/kafka/kafka.go

```diff
@@ -66,6 +67,8 @@ type AsyncProducer struct {
 	prod  sarama.AsyncProducer
 	chErr chan error
 	tag   opentracing.Tag
+	enc   encoding.EncodeFunc
+	ct    string
```

Comment:
It would be nice to rename this field to something more descriptive e.g. contentType